src/elm/Page/Community/Settings/Settings.elm

```diff
@@ -0,0 +1,122 @@
+module Page.Community.Settings.Settings exposing (Model, Msg, init, msgToString, update, view)
+
+import Api.Graphql
+import Community exposing (Community)
+import Eos exposing (Symbol)
+import Graphql.Http
+import Graphql.Operation exposing (RootQuery, RootSubscription)
+import Graphql.SelectionSet exposing (SelectionSet)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Page
+import Route exposing (Route)
+import Session.LoggedIn as LoggedIn exposing (External(..))
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> Symbol -> ( Model, Cmd Msg )
+init { shared } symbol =
+    ( initModel symbol
+    , Api.Graphql.query shared (Community.communityQuery symbol) CompletedLoad
+    )
+
+
+type alias Model =
+    { currency : String
+    , status : Status
+    }
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+type Status
+    = Loading
+    | LoadingFailed (Graphql.Http.Error (Maybe Community))
+    | Loaded Community
+
+
+type Msg
+    = CompletedLoad (Result (Graphql.Http.Error (Maybe Community)) (Maybe Community))
+
+
+initModel : Symbol -> Model
+initModel symbol =
+    { currency = "MUDA"
+    , status = Loading
+    }
+
+
+view : LoggedIn.Model -> Model -> Html Msg
+view loggedIn model =
+    case model.status of
+        Loading ->
+            div [] []
+
+        Loaded community ->
+            div []
+                [ Page.viewHeader loggedIn "Edit community" Route.Dashboard
+                , view_ community.symbol
+                ]
+
+        LoadingFailed _ ->
+            div [] []
+
+
+view_ : Symbol -> Html Msg
+view_ symbol =
+    div
+        [ class "grid my-4"
+        , style "grid-template-columns" "0 1fr 0"
+        , style "grid-template-rows" "auto"
+        , style "grid-gap" "16px"
+        ]
+        [ settingCard "Community information" "Logo, name, description" Route.Dashboard
+        , settingCard "Currency" "MUDA" Route.Dashboard
+        , settingCard "Objectives and Actions" "" Route.Dashboard
+        , settingCard "Team" "Team building" Route.Dashboard
+        , settingCard "Features" "Actions, shop" (Route.CommunitySettingsFeatures symbol)
```

Comment:
I guess we need localization there. And also `view_` is not very informative name for this function.

Reply:
I'm not really sure what to call this function. It only exists because in the actual view function we need to pattern match on the status. Any ideas?

Reply:
What about `viewSettingsList`?

Reply:
I like it :)